go-controller/cmd/ovnkube-trace/ovnkube-trace.go

```diff
@@ -501,14 +523,27 @@ func getPodInfo(coreclient *corev1client.CoreV1Client, restconfig *rest.Config,
 		podInfo.OfportNum = ovsInterfaceInformation.Ofport
 	}
 
-	podInfo.NodeExternalBridgeName, err = getNodeExternalBridgeName(coreclient, restconfig, ovnNamespace, podInfo.OvnKubePodName, cmd, podInfo.NodeName)
+	podInfo.NodeExternalBridgeName, err = getNodeExternalBridgeName(coreclient, restconfig, ovnNamespace, podInfo.OvnKubePodName, podInfo.SbCommand, podInfo.NodeName)
 	if err != nil {
 		return nil, err
 	}
 
 	return podInfo, err
 }
 
+func getSwitchPortMacAddress(coreclient *corev1client.CoreV1Client, restconfig *rest.Config, podInfo *PodInfo, ovnNamespace, portPrefix string) (string, error) {
+	tspCmd := "ovn-sbctl --no-leader-only " + podInfo.SbCommand + " --bare --no-heading --column=mac list Port_Binding " + portPrefix + podInfo.NodeName
+	ipOutput, ipError, err := execInPod(coreclient, restconfig, ovnNamespace, podInfo.OvnKubePodName, "ovnkube-node", tspCmd, "")
+	if err != nil {
+		return "", fmt.Errorf("execInPod() failed. err: %s, stderr: %s, stdout: %s, podInfo: %v", err, ipError, ipOutput, podInfo)
+	}
+	macIP := strings.Split(strings.Replace(ipOutput, "\n", "", -1), " ")
```

Comment:
add comments that show what the `ipOutput` would look like before this and why we need logic or replace its really not intuitive when reviewing... (I know other places do the same thing... still)

Reply:
done.